starboard/android/shared/player_components_factory.h

```diff
@@ -236,10 +236,11 @@ class PlayerComponentsFactory : public starboard::shared::starboard::player::
     if (creation_parameters.video_codec() != kSbMediaVideoCodecNone) {
       constexpr int kTunnelModeAudioSessionId = -1;
       constexpr bool kForceSecurePipelineUnderTunnelMode = false;
+      constexpr int kMaxInputSize = 0;
```

Comment:
This prevents configurable max input size for pass-through audio playback. The two fields above are set because tunnel mode is not enabled for pass-through, should this new field have the same restriction?